network-performance/README.md

```diff
@@ -1,202 +1,196 @@
 # Network Performance Workflow
 
-## Workflow description
+## Workflow Description
 
 This workflow defines and end-to-end benchmark test for network performance, specifically using the [uperf](https://github.com/uperf/uperf) benchmark utility. This is targeted at Kubernetes environments and includes orchestration of necessary objects via the k8s APIs.
 
-A single top-level schema provides all of the data and metadata constructs required to describe the complete list of benchmark workloads to be run, as well global and SUT (system under test) parameters that further define the environment within which the test is run and how other parallel data collections will be handled. Finally, all data and metadata from the sequence of tests are post-processed into an approproate document format and indexed into Elasticsearch.
+A single top-level schema provides all of the input constructs required to describe the benchmark workload to be run, as well global parameters that further define the environment within which the test is run and how other parallel data collections will be handled. Finally, all data and metadata from the sequence of tests are post-processed into an approproate document format and indexed into Elasticsearch.
 
-## Diagram Key
-```mermaid
-graph TD
-    schema[schema parameters]
-    engine(engine tasks)
-    plugin([plugin tasks])
-```
+## Workflow Schema
+The workflow schema defines the input parameters expected from the workflow user. The workflow author can determine how these parameters are presented to the user, independent of the subordinate plugin schemas. 
 
-## Workflow and schema breakdown
-Global parameters define metadata items that will persist through the entire workflow, as well as flags for whether specific workflow features, and therefore plugins, will be used.
+In this example workflow, we require only three input parameters: `kubeconfig`, `run_id`, and an object for `elasticsearch` connectivity.
 
 ```yaml
-global_params:
-    platform: enum
-    kubeconfig: str
-    es_server: str
-    metadata_collection: bool
-    metadata_targeted: bool
-    system_metrics_collection: bool
-    uuid: str
-    cluster_name: str
-    prom_token: str
-```
-```mermaid
-graph TD
-    global_params[global params] --> metrics_plugin([SUT metrics collection])
-    global_params --> metadata_plugin([SUT metadata collection])
-    global_params --> pod_config(k8s pod/job config)
-```
+input:
```

Comment:
Do we have a way to keep this `yaml` updated with the external file?  Perhaps have a way to re-generate the `README.md` file from a template that includes the external file?